cmd/main.go

```diff
@@ -274,41 +288,47 @@ func run(ctrlConfig spirev1alpha1.ControllerManagerConfig, options ctrl.Options,
 		GCInterval:        ctrlConfig.GCInterval,
 	})
 
-	if err = (&controllers.ClusterSPIFFEIDReconciler{
+	// FROM generated code
+
+	if err = (&controller.ClusterSPIFFEIDReconciler{
 		Client:    mgr.GetClient(),
 		Scheme:    mgr.GetScheme(),
 		Triggerer: entryReconciler,
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "ClusterSPIFFEID")
 		return err
 	}
-	if err = (&controllers.ClusterFederatedTrustDomainReconciler{
+	if err = (&controller.ClusterFederatedTrustDomainReconciler{
 		Client:    mgr.GetClient(),
 		Scheme:    mgr.GetScheme(),
 		Triggerer: federationRelationshipReconciler,
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "ClusterFederatedTrustDomain")
 		return err
 	}
-	if err = (&controllers.ClusterStaticEntryReconciler{
+	if err = (&controller.ClusterStaticEntryReconciler{
 		Client:    mgr.GetClient(),
 		Scheme:    mgr.GetScheme(),
 		Triggerer: entryReconciler,
 	}).SetupWithManager(mgr); err != nil {
 		setupLog.Error(err, "unable to create controller", "controller", "ClusterStaticEntry")
 		return err
 	}
-	if err = (&spirev1alpha1.ClusterFederatedTrustDomain{}).SetupWebhookWithManager(mgr); err != nil {
-		setupLog.Error(err, "unable to create webhook", "webhook", "ClusterFederatedTrustDomain")
-		return err
+	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
@kfox1111, you might be interested in this. Looks like a new kubebuilder convention?

Reply:
Hadn't seen that. Interesting. Thanks for the heads up.